### PR TITLE
.eb file to install GenotypeHarmonizer 1.4.14 with java 1.7.80

### PR DESCRIPTION
**Pull request by npklein**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/g/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-Java-1.7.0_80.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+name = 'GenotypeHarmonizer'
+version = '1.4.14'
+
+homepage = 'https://github.com/molgenis/systemsgenetics/wiki/Genotype-Harmonizer'
+description = """The Genotype Harmonizer is an easy to use command-line tool that allows harmonization of genotype data stored using different file formats with different and potentially unknown strands."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+easyblock = 'PackedBinary'
```
Comment (boegel) [inline easybuild/easyconfigs/g/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-Java-1.7.0_80.eb]: move this on top please, first line (and blank line above `name`)

File: easybuild/easyconfigs/g/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-Java-1.7.0_80.eb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+
+name = 'GenotypeHarmonizer'
+version = '1.4.14'
+
+homepage = 'https://github.com/molgenis/systemsgenetics/wiki/Genotype-Harmonizer'
+description = """The Genotype Harmonizer is an easy to use command-line tool that allows harmonization of genotype data stored using different file formats with different and potentially unknown strands."""
```
Comment (boegel) [inline easybuild/easyconfigs/g/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-Java-1.7.0_80.eb]: please limit to 120 chars, max:

```python
description = """The Genotype Harmonizer is an easy to use command-line tool that allows harmonization of genotype data
 stored using different file formats with different and potentially unknown strands."""
```

File: easybuild/easyconfigs/g/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-Java-1.7.0_80.eb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+easyblock = 'PackedBinary'
+
+dependencies = [('Java', '1.7.0_80')]
+
+#
+# Example URL
+# http://www.molgenis.org/downloads/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-dist.tar.gz 
+#
+download_name = 'Genotype-Harmonizer'
```
Comment (boegel) [inline easybuild/easyconfigs/g/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-Java-1.7.0_80.eb]: not used, so remove?

File: easybuild/easyconfigs/g/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-Java-1.7.0_80.eb [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+source_urls = ['http://www.molgenis.org/downloads/GenotypeHarmonizer/']
+sources = ['GenotypeHarmonizer-%s-dist.tar.gz' % version]
+
+checksums = ['0a7b041efdd7a5d7ced0ae48eb4a9e06']  #GenotypeHarmonizer-1.4.14-dist.tar.gz
```
Comment (boegel) [inline easybuild/easyconfigs/g/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-Java-1.7.0_80.eb]: move comment, no really useful? There's only one source file

File: easybuild/easyconfigs/g/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-Java-1.7.0_80.eb [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+easyblock = 'PackedBinary'
+
+dependencies = [('Java', '1.7.0_80')]
+
+#
+# Example URL
+# http://www.molgenis.org/downloads/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-dist.tar.gz 
+#
```
Comment (boegel) [inline easybuild/easyconfigs/g/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-Java-1.7.0_80.eb]: please remove this comment block, these get outdated too easily (e.g. when `--try-software-version` is used)

File: easybuild/easyconfigs/g/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-Java-1.7.0_80.eb [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+download_name = 'Genotype-Harmonizer'
+source_urls = ['http://www.molgenis.org/downloads/GenotypeHarmonizer/']
+sources = ['GenotypeHarmonizer-%s-dist.tar.gz' % version]
```
Comment (boegel) [inline easybuild/easyconfigs/g/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-Java-1.7.0_80.eb]: please use `%(version)s` template:

```python
sources = ['GenotypeHarmonizer-%(version)s-dist.tar.gz']
```

**Comment by boegel** [comment]
@npklein: some minor style remarks, looks pretty good for your first contribution! Welcome to EasyBuild!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/90fbf003e823a0f8f856 for a full test report.

**Comment by boegel** [comment]
@npklein: our test bot detected an issue with your filename:

```
AssertionError: Filename '.../GenotypeHarmonizer-1.4.14-Java-1.7.0_80.eb' of parsed easyconfig matches expected filename 'GenotypeHarmonizer-1.4.14.eb'
```

versionsuffix is not set, see https://github.com/hpcugent/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/a/ant/ant-1.9.3-Java-1.7.0_79.eb#L15 for example

File: easybuild/easyconfigs/g/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-Java-1.7.0_80.eb (hunk 2 above)
Comment (npklein) [inline easybuild/easyconfigs/g/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-Java-1.7.0_80.eb]: @boegel Is the 120 chars max per line or for the whole description?

Comment (boegel) [inline easybuild/easyconfigs/g/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-Java-1.7.0_80.eb]: per line, I didn't shorten the description, just broke it up in multiple lines, see above

Comment (boegel) [inline easybuild/easyconfigs/g/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-Java-1.7.0_80.eb]: @npklein: this one still stands?

Comment (npklein) [inline easybuild/easyconfigs/g/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-Java-1.7.0_80.eb]: @Boegel sorry was doing something wrong with my branching, I think the pull request includes this now.

File: easybuild/easyconfigs/g/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-Java-1.7.0_80.eb [hunk 7]
```diff
@@ -0,0 +1,26 @@
+toolchain = {'name': 'dummy', 'version': 'dummy'}
```
Comment (boegel) [inline easybuild/easyconfigs/g/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-Java-1.7.0_80.eb]: toolchain should go below `description`, with an empty line above/below

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/77b75ed85dc2f2043c1e for a full test report.

**Comment by boegel** [comment]
Looking good now, going on.

Congrats on your first contribution to EasyBuild @npklein!

Welcome to the wonderful world of EasyBuild... ;-)

**Pull request merged**